### ISSUE TITLE
Add an HDR panel

### ISSUE THREAD

**Issue by chenpaner** [schroef/World_Control#1]
Cool, thank god I finally found this plugin, I was looking for a plugin to control world environments with different names and this plugin did it. In actual work, I will have multiple scenes, so there will be multiple environments, I usually copy it directly, so the names will be WorldControl, WorldControl.001..., and the node group names will also be WorldControlAdvanced, WorldControlAdvanced. 001, WorldControlAdvanced.002...., I have tried many plugins but can't control the replicated node group.
This plugin is already perfect, but could you please add a panel that can preview HDR assets, I PS an imagined panel look! Thanks again for making this useful plugin!

![Snipaste_2022-10-28_21-25-51](https://user-images.githubusercontent.com/107256886/198614513-26c34333-8e5e-461d-a336-80b7db343624.png)
![Snipaste_2022-10-28_21-36-43](https://user-images.githubusercontent.com/107256886/198622105-3e7483fb-8e32-4fef-913d-c26fa68c63e6.png)

**Comment by schroef** [comment]
Does it work correctly when you have worldcontrol.001? I don't believe i have coded it so it would read alternative versions.

I made it like this, so we can control it in the 3dview without the need to have a node editor open.

I'll have a look at your idea

**Comment by schroef** [comment]
Did a vert quick test using a image_id_template. It works, i do need to style a bit better. When selecting it, it will open the image select popover, that same one as in the UV editor.  This is the easiest implementation, not sure i would need to make it more difficult. Your implementation looks a bit like the easy_hdr addon. That one works very different. it goes through a complete folder of images.
![3dview-panel-v061-preview added](https://user-images.githubusercontent.com/6923008/198707768-a12ba7f1-dfe4-45bb-9e3d-63884626b164.png)

Popover when selecting images, it looks empty, but im on old 2.83 blender on OSX. This one still has that issue rendering icons for HDR and EXR images
![3dview-panel-v061-preview added popover](https://user-images.githubusercontent.com/6923008/198708328-59c6e520-f8cc-46a6-ad2f-feb624405fa6.png)

**Comment by chenpaner** [comment]
> 当你有 worldcontrol.001 时它能正常工作吗？我不相信我已经编码了它，所以它会读取替代版本。
> 
> 我这样做了，所以我们可以在 3dview 中控制它，而无需打开节点编辑器。
> 
> 我看看你的想法

Yes, even if I change WorldControlAdvanced to any name, such as "qwe", it can still control, maybe your encoding is different from other hdr plugins, it works anyway.

**Comment by chenpaner** [comment]
> 使用image_id_template进行了垂直快速测试。它有效，我确实需要更好的风格。选择它时，它将打开图像选择弹出窗口，与UV编辑器中的弹出窗口相同。这是最简单的实现，不确定我需要使其更加困难。您的实现看起来有点像easy_hdr插件。那个工作方式非常不同。它通过一个完整的图像文件夹。 ![3dview-panel-v061-preview added](https://user-images.githubusercontent.com/6923008/198707768-a12ba7f1-dfe4-45bb-9e3d-63884626b164.png)
> 
> 选择图像时弹出框，它看起来是空的，但我在 OSX 上的旧 2.83 搅拌机上。这个仍然存在HDR和EXR图像的渲染图标问题 ![3dview-panel-v061-preview added popover](https://user-images.githubusercontent.com/6923008/198708328-59c6e520-f8cc-46a6-ad2f-feb624405fa6.png)

           Looking forward to the release of the new version you test！ In my work, I will have different kinds of hdr classified in different folders, and at the beginning of rendering I will try different hdr to find the right hdr, so I will propose For this requirement, I have also tried your EasyHDRI, but it cannot be automatically classified according to the folder, so the thumbnail list will be very large, which is inconvenient to choose.
        Regarding the problem of hdr thumbnails, I found in other plug-ins that if the size of hdr exceeds 8k, it cannot be displayed normally, you can try it to see if there is the same problem!

**Comment by chenpaner** [comment]
Looking forward to the release of the new version you test！ In my work, I will have different kinds of hdr classified in different folders, and at the beginning of rendering I will try different hdr to find the right hdr, so I will propose For this requirement, I have also tried your EasyHDRI, but it cannot be automatically classified according to the folder, so the thumbnail list will be very large, which is inconvenient to choose.
    Regarding the problem of hdr thumbnails, I found in other plug-ins that if the size of hdr exceeds 8k, it cannot be displayed normally, you can try it to see if there is the same problem!

**Comment by schroef** [comment]
>  Looking forward to the release of the new version you test！ In my work, I will have different kinds of hdr classified in different folders, and at the beginning of rendering I will try different hdr to find the right hdr, so I will propose For this requirement, I have also tried your EasyHDRI, but it cannot be automatically classified according to the folder, so the thumbnail list will be very large, which is inconvenient to choose.
>     Regarding the problem of hdr thumbnails, I found in other plug-ins that if the size of hdr exceeds 8k, it cannot be displayed normally, you can try it to see if there is the same problem!

Yeah the thumbnail issues has been worked on. In my screenshot the hdr is only 1k, but I'm on a machine from 2008 :( 
EasyHDR works better if you add separate folders. I save my HDR per folder where i got them from and then also by resolution, that works well. But that addon does not have the functionality which this addon has.

**Comment by schroef** [comment]
> > 当你有 worldcontrol.001 时它能正常工作吗？我不相信我已经编码了它，所以它会读取替代版本。
> > 
> > 我这样做了，所以我们可以在 3dview 中控制它，而无需打开节点编辑器。
> > 
> > 我看看你的想法
> 
> Yes, even if I change WorldControlAdvanced to any name, such as "qwe", it can still control, maybe your encoding is different from other hdr plugins, it works anyway.
> 
> 

So you say the panel in the 3dview still shows the controls, that should not happen if I'm correct. The way it is coded, is that it looks for a world named "world control" and then checks if it's the basic or advanced version.

I'll try to some coding perhaps today and then release a test version. See if you like it like this. I still need the think about the layout. Using this image preview changes the panel quite a lot
